_examples/golang-sse/proto/chat.gen.go

```diff
@@ -145,9 +145,16 @@ type WebRPCServer interface {
 	http.Handler
 }
 
+var (
+	annotations = map[string]map[string]string{
```

Comment:
looks like this is server-only

however I think that annotations might be useful on client-side too :)

can we move to common?

Reply:
btw: I think this var is even more useful than `var WebRPCServices` at the top of the file. Perhaps we could expose it (under different name)?